src/index.ts

```diff
@@ -195,7 +206,14 @@ const runTestsInAGroup = async (
     const startTime = new Date().valueOf();
     await runBeforeOrAfter(after, "After");
     const endTime = new Date().valueOf();
-    log(`Finished: After script in ${(endTime - startTime) / 1000} seconds\n`);
+    log(
+      `Finished: After script in ${
+        (endTime - startTime) /
```

Comment:
lets extract this into a common function outside and also write a test for it

Reply:
Also let's use this option while running our tests

https://github.com/team-black-box/good-vibes/blob/main/test/all.test.ts#L10